var/spack/repos/builtin/packages/lammps/package.py

```diff
@@ -49,17 +49,23 @@ def url_for_version(self, version):
                           'coreshell', 'dipole', 'granular', 'kspace', 'latte',
                           'manybody', 'mc', 'meam', 'misc', 'molecule',
                           'mpiio', 'peri', 'poems', 'python', 'qeq', 'reax',
-                          'replica', 'rigid', 'shock', 'snap', 'srd',
+                          'replica', 'rigid', 'shock', 'snap', 'spin', 'srd',
                           'user-atc', 'user-h5md', 'user-lb', 'user-misc',
                           'user-netcdf', 'user-omp', 'voronoi']
+    sm_levels = ('sm_20', 'sm_30', 'sm_50', 'sm_60', 'sm_70', 'sm_75')
```

Comment:
Might want to add the `CudaPackage` base class to be consistent with other packages?

Reply:
The class now extends `CudaPackage` and incorporates recent updates to the package.